dali/pipeline/operators/fused/resize_crop_mirror.h

```diff
@@ -104,8 +105,8 @@ class ResizeCropMirrorAttr {
 
     if (flag & t_crop) {
       // Crop
-      float crop_x_image_coord = spec.GetArgument<float>("crop_pos_x", ws, index);
-      float crop_y_image_coord = spec.GetArgument<float>("crop_pos_y", ws, index);
+      auto crop_x_image_coord = spec.GetArgument<float>("crop_pos_x", ws, index);
```

Comment:
Both autos look like they could be const

Reply:
Done